packages/react/src/Overlay/Overlay.tsx

```diff
@@ -227,7 +231,7 @@ const Overlay = React.forwardRef<HTMLDivElement, OwnOverlayProps>(
               ...styleFromProps,
             } as React.CSSProperties
           }
-          data-reflow-container={reflowSize && reflow ? true : undefined}
+          data-reflow-container={overflow || (reflowSize && !preventOverflow) ? true : undefined}
```

Comment:
I'm confused about this logic, wouldn't the second part be overriding the FF? or is that the intent here?

Reply:
For this, I was thinking of adding the prop outside of the FF, but putting the opt-out factor inside the feature flag. So basically someone could use the prop today, but they'd have to pass it themselves. 

if `overflow` is true, that means that the FF is enabled and a consumer would have to opt-out of it via `preventOverflow=true`. Otherwise, if the FF is not enabled, they must pass in `preventOverflow="false"` in order to get the reflow style.

Reply:
ahh I see, thanks for explaining that!